omegatrade-app/backend/app/controller/company.controller.js

```diff
@@ -0,0 +1,96 @@
+'use strict';
+const Company = require('../models/company.model')
+const { v4: uuidv4 } = require('uuid');
+
+/**
+ * Function to list all companies
+ * @method GET
+ */
+exports.getList = async function (req, res) {
+    try {
+        await Company.getAll(function (err, data) {
+            if (err)
+                return res.json({ success: false, message: "Error occured while fetching all Company" });
```

Comment:
type in work occured here and elsewhere:  `Error occurred`

Reply:
Error message has been changed to generic `something went wrong`